index.bs

```diff
@@ -213,10 +220,15 @@ spec: SP800-800-63r3; urlPrefix: https://pages.nist.gov/800-63-3/sp800-63-3.html
         text: something you know; url: af
         text: something you have; url: af
         text: something you are; url: af
+
 spec: webidl; urlPrefix: https://heycam.github.io/webidl
     type: dfn;
         text: get a copy of the bytes held by the buffer source; url: dfn-get-buffer-source-reference
 
+spec: UTR29
+    urlPrefix: https://unicode.org/reports/tr29/
+        type: dfn; for:/; url: Grapheme_Cluster_Boundaries; text: grapheme cluster
+
```

Comment:
Why is this biblio separate from the one at the end of the file?

Reply:
because the UTR29 at end of file is a biblio entry, while the above is a cross-spec link specifically for "grapheme cluster"

Reply:
Whoops, I put this on the wrong diff hunk...